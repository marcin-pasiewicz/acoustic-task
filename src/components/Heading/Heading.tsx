import React, { ReactElement } from 'react'

interface Props {
  value: string;
}

export default function Heading({value}: Props): ReactElement {
  return (
    <h1>
      {value}
    </h1>
  )
}
