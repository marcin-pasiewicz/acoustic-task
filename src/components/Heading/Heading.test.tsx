import React from 'react';
import { render } from '@testing-library/react';
import Heading from './Heading';

describe('<Heading/>', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Heading value="Awesome heading" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
