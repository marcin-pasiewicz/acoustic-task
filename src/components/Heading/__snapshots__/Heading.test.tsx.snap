// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading/> renders correctly 1`] = `
<DocumentFragment>
  <h1>
    Awesome heading
  </h1>
</DocumentFragment>
`;
